src/js/components/AppListComponent.jsx

```diff
@@ -126,17 +125,9 @@ var AppListComponent = React.createClass({
 
     if (props.filterTypes != null && props.filterTypes.length > 0) {
       appsSequence = appsSequence.filter(function (app) {
-        let appTypeIndex = 0;
-        if (app.container != null && app.container.type != null) {
-          appTypeIndex = AppTypes.indexOf(app.container.type);
-          if (appTypeIndex === -1) {
-            return false;
-          }
-        }
-
         /* Use .every for an INTERSECTION instead of UNION */
```

Comment:
@aldipower that was me. I left it there because, at the time of writing, I did not have a final answer about how our filters should actually work when stacked. We can remove the comment, sure.